### PR TITLE
Prefer yarn

### PR DESCRIPTION
**Pull request by drassaby**
We should all use one dependency manager. yarn has faster installation times and is generally considered more secure than npm. It's also on Travis by default, so I suggest we use that

### CODE REVIEW

**Review by verdabursal — CHANGES_REQUESTED** [review]
spun up locally with yarn and verified that this works. only change needed is a note abt `yarn install` in the readme

File: README.md [hunk 1]
```diff
@@ -20,20 +32,20 @@ The website may take a few minutes to deploy. After waiting a few minutes, you s
 
 Note: eventually all deployment will be done during CI/CD and no humans will have push access.
```
[the remark this answers is not in the document]
Comment (drassaby) [reply README.md:34]: Sure 👍

**Review by verdabursal — DISMISSED** [review]
lgtm!

**Review by verdabursal — APPROVED** [review]
verified that app starts up locally

**Pull request merged**